var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -269,6 +269,11 @@ def write_library_dirs(f, dirs):
                 f.write("libraries = {0}\n".format(lapackblas_lib_names))
                 write_library_dirs(f, lapackblas_lib_dirs)
                 f.write("include_dirs = {0}\n".format(lapackblas_header_dirs))
+                f.write(
+                    "symbol_suffix = {0}\n".format(
+                        spec["openblas"].variants["symbol_suffix"].value
+                    )
+                )
```

Comment:
This correctly treats the value `"none"` (which is a string, not `None`):
```suggestion
                symbol_suffix = spec["openblas"].variants["symbol_suffix"].value
                if symbol_suffix != "none":
                    f.write("symbol_suffix = {0}\n".format(symbol_suffix))
```